Autopilot/AttitudeManager/nbproject/configurations.xml

```diff
@@ -193,8 +196,9 @@
         <property key="use-cci" value="false"/>
         <property key="use-iar" value="false"/>
       </C30>
-      <C30-AS>
-      </C30-AS>
+      <C30-AR>
+        <property key="additional-options-chop-files" value="false"/>
```

Comment:
If you add new files they keep them in the project settings,  so you
sometimes need to include them. Otherwise you have to add everything to the
project manually.

Trouble is it also makes changes based on what operating system you are on
and where your working directory is.

It's a problem we had before and kind of just said 'screw it'.

On Jul 20, 2016 5:27 PM, "Serge Babayan" <notifications@github.com> wrote:

> In Autopilot/AttitudeManager/nbproject/configurations.xml
> <https://github.com/UWARG/PICpilot/pull/56#discussion_r71610128>:
>
> > @@ -193,8 +196,9 @@
> >          <property key="use-cci" value="false"/>
> >          <property key="use-iar" value="false"/>
> >        </C30>
> > -      <C30-AS>
> > -      </C30-AS>
> > +      <C30-AR>
> > +        <property key="additional-options-chop-files" value="false"/>
>
> Can we not commit these config files? Maybe add the project config folder
> to the git ignore for now. They still need to be within the repo but theres
> no pointing in updating them any further.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/UWARG/PICpilot/pull/56/files/cb2f55f9bc7d8149f5aa8fb970401ef6408be50b#r71610128>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC4dF_OxPCq2KM1yc_EoLoVLjWODoATHks5qXpLUgaJpZM4JRHQr>
> .
>